test/e2e/util.go

```diff
@@ -0,0 +1,448 @@
+//+build e2e
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"reflect"
+	"testing"
+	"time"
+
+	configv1 "github.com/openshift/api/config/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	operatorv1alpha1 "github.com/openshift/external-dns-operator/api/v1alpha1"
+
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/aws/aws-sdk-go/service/route53"
+)
+
+var defaultExtDNSTemplate = operatorv1alpha1.ExternalDNS{
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      "test-extdns",
+		Namespace: "publish-external-dns",
```

Comment:
ExternalDNS custom resource is a cluster level object. No need to specify a namespace.

Reply:
fixed, thanks